### PR TITLE
Group callbacks config into separate files

### PR DESCRIPTION
**Pull request by yu-xiang-wang**
write all callbacks config in just one file is not very clear, i have seperate different callback into their own file, and so if one wants to change some callback, for example model_checkpoint, he can look at model_checkpoint.yaml and just modify the parameters he want in callbacks/default.yaml

### CODE REVIEW

**Comment by ashleve** [comment]
@yu-xiang-wang Thank you for the PR! I like these changes.

File: configs/callbacks/early_stopping.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.EarlyStopping.html#pytorch_lightning.callbacks.EarlyStopping
```
Comment (ashleve) [inline configs/callbacks/early_stopping.yaml:1]: I think we can shorten these links?
```suggestion
# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.EarlyStopping.html
```

File: configs/callbacks/model_checkpoint.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.ModelCheckpoint.html#pytorch_lightning.callbacks.ModelCheckpoint
```
Comment (ashleve) [inline configs/callbacks/model_checkpoint.yaml:1]:
```suggestion
# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.ModelCheckpoint.html
```

File: configs/callbacks/model_summary.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.RichModelSummary.html#pytorch_lightning.callbacks.RichModelSummary
```
Comment (ashleve) [inline configs/callbacks/model_summary.yaml:1]: ### 
```suggestion
# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.RichModelSummary.html
```

File: configs/callbacks/rich_progress_bar.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.RichProgressBar.html#pytorch_lightning.callbacks.RichProgressBar
```
Comment (ashleve) [inline configs/callbacks/rich_progress_bar.yaml:1]:
```suggestion
# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.RichProgressBar.html
```

File: configs/callbacks/early_stopping.yaml (hunk 1 above)
Comment (yu-xiang-wang) [reply configs/callbacks/early_stopping.yaml:1]: Yes, that will be more clear, i will change this and add some comment to every argument

**Review by ashleve — APPROVED** [review]

**Comment by ashleve** [comment]
Hi @yu-xiang-wang, this looks good, thank you! :rocket:

**Pull request merged**